src/components/seeds/database/index.tsx

```diff
@@ -200,10 +202,45 @@ export default function Database(props: DatabaseProps): JSX.Element {
   const [unfilteredResults, setUnfilteredResults] = useState<SearchResponseElement[] | null>();
   const [selectSeedBankForImportModalOpen, setSelectSeedBankForImportModalOpen] = useState<boolean>(false);
   const [openImportModal, setOpenImportModal] = useState<boolean>(false);
+  const [showDefaultSystemSnackbar, setShowDefaultSystemSnackbar] = useState(false);
+  const { userPreferences } = useOrganization();
+  const weightUnitsEnabled = isEnabled('Weight units');
+  const snackbar = useSnackbar();
+
+  useEffect(() => {
+    const showSnackbar = () => {
+      snackbar.pageInfo(
+        strings.formatString<any>(
+          strings.CHANGE_DEFAULT_WEIGHT_SYSTEM,
+          <Link to={APP_PATHS.MY_ACCOUNT}>{strings.MY_ACCOUNT}</Link>
+        ),
+        '',
+        {
+          label: strings.GOT_IT,
+          apply: async () => {
+            await PreferencesService.updateUserPreferences({ defaultWeightSystemAcknowledgedOnMs: Date.now() });
+            reloadPreferences();
+          },
```

Comment:
unrelated to this PR but a request if possible.. if we could move this as a utility hook somewhere and use it in all those places that have a got-it button to set a preference and reload preference (time zone, unit, etc)...
example:
```
useGetAcknowledgedPreferenceContent(preferenceNames: string[]) {
  const { reloadPreferences } = useOrganization();
return {
  label: strings. GOT_IT,
  apply: async () => {
    await Promise.allSettled(preferenceNames.map(name => PreferencesService.upateUserPreference(name, Date.now()));
    reloadPreferences();
  }
}
```
np if we want to do this later...

Reply:
I just tried doing this, but not all the setting preferences have the reload after the update. Some of them have a validation to update or not. For example: 
```
(      if (userUnit.updated) {
        reloadPreferences();
      } ) 
```
And another problem is that I can't call a hook from a useEffect, and we are updating froma useEffect

Reply:
Ok got it. No worries.

Reply:
Thanks for trying this out,